src/main/java/com/locallampoon/fiveh/data/rooms.xml

```diff
@@ -211,7 +211,7 @@ does not appear to be another exit from this room other than the door to the (WE
         <roomItems type="array">
             <item>CPR Dummy</item>
         </roomItems>
-        <monster questItem="Ghost's Key" weaknesses="Holy Salt">Ghost</monster>
+        <monster questItem="Ghost Key" weaknesses="Holy Salt">Ghost</monster>
```

Comment:
Looks like there are duplicate Holy Salt in both the Dining Room and Pantry

Reply:
Removed the Dining Room salt.